src/ResourceManager/Aks/Commands.Aks.Test/Commands.Aks.Test.Netcore.csproj

```
@@ -24,7 +24,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="YamlDotNet" Version="4.2.4" />
+    <PackageReference Include="YamlDotNet" Version="5.0.0" />
```

```suggestion
    <PackageReference Include="YamlDotNet.Signed" Version="5.0.0" />
```
There isn't really a reason for us to reference the unsigned version; even though this is just the test project. It just makes it easier when the compiler resolves the references.